db/view/view.cc

```diff
@@ -3741,6 +3741,8 @@ future<> view_building_worker::build_views_range(table_id base_id, dht::token_ra
             
             try {
                 utils::get_local_injector().inject("view_building_worker_pause_before_consume", 5min, _view_building_as).get();
+                utils::get_local_injector().inject("view_building_worker_wait_before_consume", utils::wait_for_message(5min)).get();
```

Comment:
Can we reuse/alter view_building_worker_pause_before_consume here instead of adding a second injection in the same place?